query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -112,3 +113,66 @@ func (PushDownFilterRule) Rewrite(pn plan.Node) (plan.Node, bool, error) {
 
 	return pn, true, nil
 }
+
+// ReadTagKeysRule matches 'ReadRange |> keys() |> keep() |> distinct()'.
+// The 'from()' must have already been merged with 'range' and, optionally,
+// may have been merged with 'filter'.
+// If any other properties have been set on the from procedure,
+// this rule will not rewrite anything.
+type ReadTagKeysRule struct{}
```

Comment:
Would you mind renaming the rule to `PushDownTagKeysRule` just for consistency?

Reply:
Done.